### DISCUSSION TITLE
Default data version is now v4!

### DISCUSSION THREAD

**Discussion by VeruGHub** [VeruGHub/easyclimate#46] [Announcements]
Since we relased easyclimate [version 0.2](https://verughub.github.io/easyclimate/news/index.html) the default data that users download is v4 unless the user specifies `version = 3`. Please, check [this document](https://figshare.com/articles/online_resource/Description_and_Evaluation_of_Downscaled_Daily_Climate_Data_Version_4/22962671/1) for information on how v4 is calculated and differences with v3.

**Comment by kgerb** [comment]
Awesome! Do you have any idea when the datasets will be updated to 2023?
